test/GithubApi.Repositories.test.js

```diff
@@ -0,0 +1,53 @@
+const { StatusCodes } = require('http-status-codes');
+const { expect } = require('chai');
+const axios = require('axios');
+const chai = require('chai');
+const chaiSubset = require('chai-subset');
+const md5 = require('md5');
+
+chai.use(chaiSubset);
+
+const urlrepos = 'https://api.github.com/repos/aperdomob/jasmine-json-report';
+
+describe('Repositories GET', () => {
+  it('Consume GET Service on GITHUB API', async () => {
+    const response = await axios.get('https://api.github.com/users/aperdomob');
+    expect(response.status).to.equal(StatusCodes.OK);
+    expect(response.data.name).to.equal('Alejandro Perdomo');
+    expect(response.data.company).to.equal('Perficient Latam');
+    expect(response.data.location).to.equal('Colombia');
+  });
+
+  it('Finding by Hypermedia', async () => {
+    const response = await axios.get('https://api.github.com/users/aperdomob/repos');
+    const repo = response.data;
+    function isjasmine(nombre) {
+      return nombre.name === 'jasmine-json-report';
+    }
+    const jasmine = repo.find(isjasmine);
+    expect(jasmine.name).to.equal('jasmine-json-report');
+    expect(jasmine.private).to.equal(false);
+    expect(jasmine.description).not.equal(null);
+  });
+  it('Downloading by zip', async () => {
+    const response = await axios.get(`${urlrepos}/zipball/master`);
```

Comment:
the repository response of the previous test (which you store in the jasmine variable) can be used to get this URL (if you use `repo.url`):
![image](https://user-images.githubusercontent.com/10209180/176711611-92f3d2a4-def3-44be-8bbf-837f36507eb0.png)